arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -133,10 +177,7 @@
         ],

```

Comment:
Please add descriptions
delete: "Deletes a container registry in Microsoft Azure."
resourceGroup: "The name of the resource group within the user's subscription."
registryName: "The name of the container registry."